src/primitives.jl

```diff
@@ -179,6 +179,13 @@ end
 dropnull(X::NullableVector) = X.values[!X.isnull]    # -> Vector
 
 """
+    dropnull!(X::AbstractVector)
+
+Remove the null entries of `X` inplace.
```

Comment:
Should explain that 1) it's a no-op for non-nullable vector, and 2) it returns a `Vector{T}` for `AbstractVector{<:Nullable{T}}`.